.travis.yml

```diff
@@ -1,11 +1,13 @@
 # The testing matrix **for simp-packer** only considers ruby/puppet versions
 # supported by SIMP:
 # ------------------------------------------------------------------------------
-#  release    pup   ruby   eol
-# SIMP 6.0    4.8   2.1.9  TBD***
-# SIMP 6.2    4.10  2.1.9  TBD
+#  release    pup   ruby     eol
+# SIMP 6.0    4.8   2.1.9    TBD***
```

Comment:
SIMP 6.1?

Reply:
@lnemsick-simp  :ballot_box_with_check: 

Thid matrix documentation uses the same format as our other projects' pipelines, so if SIMP 6.1 re-uses the same component versions as SIMP 6.0, only SIMP 6.0 is listed.  Perhaps that isn't as clear when it's only SIMP versions listed, so I've tacked a `+` at the end.